dev/com.ibm.ws.webcontainer/src/com/ibm/ws/webcontainer/osgi/webapp/WebApp.java

```diff
@@ -1057,27 +1063,18 @@ protected void addClassToHandlesTypesStartupSet(String targetClassName, Set<Clas
       
       boolean enableTrace = ( com.ibm.ejs.ras.TraceComponent.isAnyTracingEnabled() && logger.isLoggable (Level.FINE) );
 
-      int servletSpecLevel = com.ibm.ws.webcontainer.osgi.WebContainer.getServletContainerSpecLevel();
+ 
       //check if excluded because the subclasses includes this... and could be more than we want
       //might need to exclude something else like javax.* or java*
-      if(servletSpecLevel <=  com.ibm.ws.webcontainer.osgi.WebContainer.SPEC_LEVEL_40){
-          if ((targetClassName.startsWith("java.")) || (targetClassName.startsWith("javax."))) {
-                  if ( enableTrace ) {
-                        logger.logp(Level.FINE, CLASS_NAME, methodName,
-                                    "Internal class, {0}, is not added to the ServletContainerInitializers [Servlet Spec Level: {2}] in the application: {1}.",
-                                    new Object[] { targetClassName, this.config.getDisplayName(), servletSpecLevel});
-                  }
-                      return;
-          }
-      } else {
-              if ((targetClassName.startsWith("java.")) || (targetClassName.startsWith("jakarta."))) {
-                  if ( enableTrace ) {
-                        logger.logp(Level.FINE, CLASS_NAME, methodName,
-                                    "Internal class, {0}, is not added to the ServletContainerInitializers [Servlet Spec Level: {2}] in the application: {1}.",
-                                    new Object[] { targetClassName, this.config.getDisplayName(), servletSpecLevel });
-                  }
-                      return;
-              }
+      if (targetClassName.startsWith("java")) {
+        if ((targetClassName.charAt(4) == '.') || (targetClassName.startsWith("javax."))) {
+            if ( enableTrace ) { 
+                logger.logp(Level.FINE, CLASS_NAME, methodName,
+                            "Internal class, {0}, is not added to the ServletContainerInitializers in the application: {1}",
+                            new Object[] { targetClassName, this.config.getDisplayName() });
+            }
+            return;
+        }
```

Comment:
Reverted to original code.

Reply:
can it be just:

if ((targetClassName.startsWith("java.")) || (targetClassName.startsWith("jakarta."))) {

Reply:
I don't see any issue with that.  Updating...